packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -196,6 +212,23 @@ export class Tooltip {
     this.hostElement.addEventListener('mouseleave', () => {
       this.hideTooltip();
     });
+    this.hostElement.addEventListener('focusin', () => {
+      if (this.interactive) {
+        clearTimeout(this.hideTooltipTimeout);
+      }
+    });
+    this.hostElement.addEventListener('focusout', () => {
+      this.hideTooltip();
+    });
```

Comment:
Expect of the event name it is duplicated code. Please avoid code duplications.

Reply:
Does this mean that this code fragment should be deleted without replacement or are additional code modifications required?